code/game/objects/structures/statues.dm

```diff
@@ -7,25 +7,81 @@
 	icon = 'icons/obj/statue.dmi'
 	icon_state = ""
 	density = 1
-	anchored = 1
+	anchored = 0
 	var/hardness = 1
 	var/oreAmount = 7
 	var/mineralType = "metal"
-	var/last_event = 0
-	var/active = null
-	var/spam_flag = 0
 
 /obj/structure/statue/Destroy()
 	density = 0
 	..()
 
-/obj/structure/statue/attackby(obj/item/weapon/W, mob/user)
-	hardness -= W.force/100
-	user << "You hit the [name] with your [W.name]!"
-	CheckHardness()
+/obj/structure/statue/attackby(obj/item/weapon/W, mob/living/user as mob)
+	add_fingerprint(user)
+	user.changeNext_move(CLICK_CD_MELEE)
+	if(istype(W, /obj/item/weapon/wrench))
+		if(anchored)
+			playsound(src.loc, 'sound/items/Ratchet.ogg', 100, 1)
+			user.visible_message("<span class='notice'>[user] is loosening the [name]'s bolts...</span>", \
+								 "<span class='notice'>You are loosening the [name]'s bolts...</span>")
+			if(do_after(user,40))
+				if(!src) return
+				if(!anchored) return
+				user.visible_message("<span class='notice'>[user] loosened the [name]'s bolts!</span>", \
+									 "<span class='notice'>You loosened the [name]'s bolts!</span>")
+				anchored = 0
+		else if(!anchored)
+			if (!istype(src.loc, /turf/simulated/floor))
+				user.visible_message("<span class='danger'>A floor must be present to secure the [name]!</span>")
+				return
+			playsound(src.loc, 'sound/items/Ratchet.ogg', 100, 1)
```

Comment:
He can't, actually, there's a check  so you cant fasten them in space (which should use isinspace() instead of this istype, but w/e), and it shouldnt play if its not fastened.